source/manual/alerts/email-alert-api-unprocessed-work.html.md

```diff
@@ -0,0 +1,57 @@
+---
+owner_slack: "#govuk-2ndline"
+title: 'Email Alert API: Unprocessed work'
+section: Icinga alerts
+subsection: Email alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-10-05
+review_in: 6 months
+---
+
+> Sometimes we lose work due to [a flaw with the Sidekiq queueing system](https://github.com/mperham/sidekiq/wiki/Problems-and-Troubleshooting#my-sidekiq-process-is-crashing-what-do-i-do). At 30 minute intervals a job, [RecoverLostJobsWorker], will try to requeue work that has not been processed [within an hour](https://github.com/alphagov/email-alert-api/blob/2f3931ac1ca25fe8c79b2405af98d1de55e1d47b/app/workers/recover_lost_jobs_worker/unprocessed_check.rb#L13).
```

Comment:
This feels a bit strange to be a blockquote?

Reply:
After re-reading, I'm inclined to agree. I've moved and rewritten this paragraph to go after the list of alerts.